readme.md

```diff
@@ -152,13 +152,13 @@ Let's collaboratively create a `Person` class!
     end
   end
   ```
-
+  Note: there is also `attr_writer` that would allow instance variables to be modified, but not accessed. You will rarely need to use it.
 </details>
 
 --------
 
 <details>
-  <summary><strong>Let's give our <code>Person</code> class a <code>say_name</code> method that prints his/her name to the console.</strong></summary>
+  <summary><strong>Let's give our <code>Person</code> class a <code>say_name</code> method that prints his/her name to the terminal.</strong></summary>
 
```

Comment:
curious why you made this change. technically accurate though: http://askubuntu.com/questions/506510/what-is-the-difference-between-terminal-console-shell-and-command-line